src/js/tags-input.js

```diff
@@ -28,6 +28,11 @@ const onInput = (e) => {
       tagify.settings.whitelist.splice(0, whitelist.length, ...whitelist)
       tagify.loading(false).dropdown.show.call(tagify, value); // render the suggestions dropdown
     })
+    .catch(err => {
+      if (err.name === 'AbortError') return;
+
+      throw err;
+    });
```

Comment:
I think the proper way would be to _debounce_ the request instead of hiding an error. We can send the request X miliseconds after the last keystroke so they won't be send many times. Something like: https://codepen.io/jpkempf/pen/MbePGB

```
let timeout;

timeout = setTimeout(() => {
  clearTimeout(timeout);
  // do the call
}, 200);
```

Reply:
@z-x Fixed.